speech/unit_tests/test_sample.py

```diff
@@ -42,6 +42,39 @@ def test_content_and_source_uri(self):
             self._make_one(content='awefawagaeragere',
                            source_uri=self.AUDIO_SOURCE_URI)
 
+    def test_content_to_stream(self):
+        from google.cloud.speech.encoding import Encoding
+
+        content = 'testing 1 2 3 4'
+        sample = self._make_one(content=content,
+                                encoding=Encoding.FLAC,
+                                sample_rate=self.SAMPLE_RATE)
+
+        self.assertIsNotNone(getattr(sample.stream, 'read', None))
+
+    def test_content_is_file_like(self):
+        from io import BytesIO
+        from google.cloud.speech.encoding import Encoding
+
+        test_bytes = b'testing 1 2 3 4'
+        content = BytesIO(test_bytes)
+        sample = self._make_one(content=content,
+                                encoding=Encoding.FLAC,
+                                sample_rate=self.SAMPLE_RATE)
+        self.assertEqual(sample.content, test_bytes)
+
+    def test_bytes_converts_to_file_like_object(self):
```

Comment:
So depending on how the `stream` property ends up working, this could be removed.

I had setup `stream` to check if `content` was set and then just convert that to a file-like object through `BytesIO`